sympy/combinatorics/free_group.py

```diff
@@ -127,6 +127,7 @@ class FreeGroup(DefaultPrinting):
     """
     is_associative = True
     is_group = True
+    is_fp_group = True
```

Comment:
To me ``is_fp_group`` seems Ok. I am not a fan of ``is_FpGroup`` instead of using ``isinstance`` though.